app/views/admin/extensions/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with(url: toggle_admin_extensions_path, method: :post, local: true) do |f| %>
+  <div class="crayons-card p-4 flex flex-col gap-4">
+    <% @extensions.each do |extension| %>
+      <div>
+        <div class="crayons-field crayons-field--checkbox">
+          <%= f.check_box extension.feature_flag_name, class: "crayons-checkbox", checked: extension.enabled? %>
+          <%= f.label extension.feature_flag_name, extension.name, class: "crayons-field__label" %>
+        </div>
+        <p class="pl-7 crayons-field__description">
+          <%= sanitize extension.description %>
+        </p>
+      </div>
+    <% end %>
+    <button class="c-btn c-btn--primary">Save</button>
```

Comment:
```suggestion
    <button class="c-btn c-btn--primary self-start">Save</button>
```

(suggestion / not blocking) - I think it looks a bit funky that the button takes up the full width of the page. If we add this class it should just occupy the "normal" size for the text inside, e.g.:

![Save button aligned to start with usual padding](https://user-images.githubusercontent.com/20773163/158801029-057ed1f3-b181-421d-9073-a5b9111f466b.png)